quelpa.el

```diff
@@ -64,6 +64,12 @@ the `:upgrade' argument."
   :group 'quelpa
   :type 'boolean)
 
+(defcustom quelpa-force-upgrade-p nil
```

Comment:
As noted in another comment, this variable and `quelpa--use-current-state-p` seem to overlap or conflict.

Reply:
Please refer to my other comment for detail explain for this